test/server/action_creators_spec.js

```diff
@@ -89,4 +88,46 @@ describe('server action creators', () => {
       done();
     });
   });
+
+  it('loadItems', (done) => {
+    let dispatched;
+    function dispatch(action) {
+      dispatched = action;
+    }
+    const expected = {
+      type: constants.SET_STATE,
+      state: {
+        items: []
+      }
+    };
+
+    actions.loadItems()(dispatch).then(() => {
+      expect(dispatched).to.deep.equal(expected);
+      done();
+    });
+  });
+
+  xit('addOrder', () => {
```

Comment:
Hmm.. we could create a new Order stub as a constructor (vs. object like above) for this test and proxyquire actions by test..? Eh not sure...